tests/scripts/thread-cert/Cert_9_2_04_ActiveDataset.py

```diff
@@ -79,6 +83,8 @@ def test(self):
         )
         self.simulator.go(3)
         self.assertEqual(self.nodes[LEADER].get_network_name(), 'GRL')
+        self.nodes[COMMISSIONER].send_mgmt_active_get()
+        self.simulator.go(3)
```

Comment:
any reason for this change?

Reply:
align with test plan step 4: Commissioner sends MGMT_ACTIVE_GET.req